### PR TITLE
Better Firefox installation instructions + Firefox for Android

### PR DESCRIPTION
**Pull request by Samq64**
This might still need some work. The installation process is not very user-friendly and tricky to explain.

### CODE REVIEW

**Comment by WorldLanguages** [comment]
There's a few typos

**Comment by WorldLanguages** [comment]
I still think we could do better on this page, the normal install button for average users can only be found in the navbar.

Maybe we could add big Chrome Web Store and Firefox Addons images just like the extension repo Readme 🤔

I know this is off topic

**Comment by Samq64** [comment]
> Maybe we could add big Chrome Web Store and Firefox Addons images just like the extension repo Readme 🤔

I stuck them at the top of the page.

File: content/docs/getting-started/installing.md [hunk 1]
```diff
@@ -97,3 +100,41 @@ To reload the extension when testing, click the "Reload" button on the extension
...
+```
+
+The extension should install and automatically reload when changes are made, otherwise close the Firefox app and re-run the command.
+
+#### Inspecting the extension
+1. On desktop Firefox type `about:debugging` into the address bar.
+1. Click "Enable USB Devices".
+1. Click connect on the device in the sidebar.
+
+The extension card should show up similar a desktop one, if not restart desktop Firefox and try again.
```
Comment (WorldLanguages) [inline content/docs/getting-started/installing.md:140]: Maybe rephrase this, it's weirdly written and I'm not sure what it refers to.

File: content/docs/getting-started/installing.md [hunk 2]
```diff
@@ -97,3 +100,41 @@ To reload the extension when testing, click the "Reload" button on the extension
...
+
+1. Download and extract the Android SDK Platform Tools ([Windows](https://dl.google.com/android/repository/platform-tools-latest-windows.zip), [MacOS](https://dl.google.com/android/repository/platform-tools-latest-darwin.zip), [Linux](https://dl.google.com/android/repository/platform-tools-latest-linux.zip)).
+1. Add the folder to the PATH environment variable so it can be used anywhere.
+1. Install web-ext with npm: `npm install --global web-ext`.
+
+##### Android
+
+1. In the Android settings app, open the about page and tap the build number 7 times.
+1. Navigate to the "Developer options" page and enable "USB debugging".
+1. Open the Firefox app and enable "Remote debugging via USB" in its settings.
```
Comment (WorldLanguages) [inline content/docs/getting-started/installing.md:121]: We could clarify this guide does not consider Firefox Beta nor Nightly channels.

```suggestion
1. Install the standard Firefox browser (not beta nor nightly channels) from the Play Store.
1. Open the Firefox app and enable "Remote debugging via USB" in its settings.
```

**Review by WorldLanguages — APPROVED** [review]

**Comment by Samq64** [comment]
I made some more changes. Do you still approve?

**Comment by WorldLanguages** [comment]
I still don't see the motivation of using `web-ext`. What's the benefit?

**Comment by Samq64** [comment]
> I still don't see the motivation of using `web-ext`. What's the benefit?

Mainly that it keeps the extension settings separate from the store release. It also seems to solve the CSS caching issues but I could be wrong.

**Comment by hueychen27** [comment]
> I still don't see the motivation of using `web-ext`. What's the benefit?

I should have mentioned this, so now I will. 
The top 3 benefits:

1. It is a convenient tool used for extension development as it automatically loads the extension to a browser with one command. This means that `web-ext` is a much improved way of loading an extension with `about:debugging`.
2. It has features like auto-reloading and linting. Auto-reloading is useful not directly in the webpage but in the extension popup and background services.
3. Easy building in case you ever want to self distribute an extension.

**Comment by Samq64** [comment]
I reverted the web-ext instructions for desktop. It can be PRed separately.

**Review by WorldLanguages — APPROVED** [review]

**Comment by hueychen27** [comment]
@Samq64 Title change for this pr
